app/views/groups/show.html.erb

```diff
@@ -0,0 +1,31 @@
+
+
+ <div class="category-show-container">
+
+    <%= link_to groups_path do %>
+      <i class="fa-solid fa-circle-chevron-left back-icon"></i>
+    <% end %>
+
+   <div class="category-show">
+      <p class="category-icon"> <%= @group.icon %> </p>
+      <p> <%= @group.name.capitalize %> </p>
+   </div>
+
+    <div>
+        <% @group.entities.each do |entity| %>
+
+        <div class="category-detail">
+             <div class="category-amount">
+                <p> <%= entity.name.capitalize %></p>
+                <p> $<%= entity.amount %></p>
+             </div>
+              <p> <%= entity.created_at%> </p>
+          </div>
+          <% end %>
+        <div class="category-amount total-amount">
+           <p>TOTAL PAYMENT </p>
```

Comment:
- [ ] I think it is very important to follow the requirements and the design guidelines. We shall always develop a project based on the requirements of the client. We were told to display the total amount of all transactions for each category at the top of the page NOT at the bottom.